root.spec

```diff
@@ -22,6 +22,21 @@ Requires: freetype
 
 %define keep_archives true
 
+%define subpackageDebug true
+
+%if "%{?subpackageDebug:set}" == "set"
+%define envOptions USER_CXXFLAGS="-fdebug-prefix-map=%{cmsroot}=%{installroot} -fdebug-prefix-map=%{instroot}=%{installroot} -g"
+%endif
+# For non-linux archs disable subpackageDebug otherwise add dwz.file build dependencies.
+%if "%{?subpackageDebug:set}" == "set"
+%if %islinux
+BuildRequires: file
+%else
+%undefine subpackageDebug
+%endif # islinux
+%endif # subpackageDebug
+
+
```

Comment:
See scram-project-build.spec. I copied the pattern as @smuzaffar suggested. Either reject the pull request or take it as is.